apiary.apib

```diff
@@ -3289,6 +3289,144 @@ Download a credit note in a specific format.
             + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
             + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
 
+## Subscriptions [/subscriptions]
+
+### subscriptions.list [GET /subscriptions.list]
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string], optional)
+            + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional) - Filter on subscriptions created from an invoice
```

Comment:
the subscriptions aren't created by the invoice, it's the other way around

```suggestion
            + invoice_id: `60f03745-4289-4ae7-8c74-d797860bc887` (string, optional) - Find subscriptions that generated the given invoice
```

Reply:
Not sure this is correct. It is possible to create a subscription from deal/invoice.
We persist the **parent** invoice in this case, not the resulting invoices of that subscription

Reply:
Let's update the naming to make it clearer that this is indeed filtering on related invoices (and not the actual parent invoice)